pom.xml

```diff
@@ -1525,13 +1525,26 @@
                     <version>${spotless-plugin.version}</version>
                     <configuration>
                         <java>
-                            <removeUnusedImports />
+                            <removeUnusedImports/>
+                            <!--<forbidWildcardImports /> fix with rewrite: https://docs.openrewrite.org/recipes/java/removeunusedimports -->
                         </java>
+                        <lineEndings/>
+                        <pom>
+                            <sortPom/>
+                        </pom>
+                        <upToDateChecking>
+                            <enabled>true</enabled>
```

Comment:
this is kind of caching. We should follow and not try to outsmart big brother maven. 

- https://github.com/apache/maven-parent/blob/6e19d396b44e56affe38a7daf8a71676dc4e46ae/pom.xml#L1235